src/golang/cmd/server/request/integration.go

```diff
@@ -43,7 +43,11 @@ func ParseIntegrationConfigFromRequest(r *http.Request) (string, map[string]stri
 
 // isUserOnlyIntegration returns whether the specified service is only accessible by the user.
 func isUserOnlyIntegration(svc integration.Service) bool {
-	userSpecific := []integration.Service{integration.GoogleSheets, integration.Github}
+	userSpecific := []integration.Service{
+		integration.GoogleSheets,
+		integration.Github,
+		integration.Conda,
```

Comment:
I actually think it's possible in the future for users to share the Conda env created. But since now we're focusing on OSS, there is only one user, so this seems irrelevant for now.